libs/constants/src/lib/constants.ts

```diff
@@ -0,0 +1,3 @@
+export function constants(): string {
```

Comment:
This file should be exporting all of our constants like this:
`export * as CITIES from './cities';`
So can remove the constants function and replace it with multiple exports.